packages/app-mobile/components/NotesBar.tsx

```diff
@@ -0,0 +1,282 @@
+import * as React from 'react';
+import { View, Text, TextInput, TouchableOpacity, FlatList } from 'react-native';
+import { State } from '@joplin/lib/reducer';
+import { themeStyle } from './global-style';
+const { connect } = require('react-redux');
+const Icon = require('react-native-vector-icons/Ionicons').default;
+const { _ } = require('@joplin/lib/locale');
+import { Style } from './global-style';
+import Note from '@joplin/lib/models/Note';
+import NotesBarListItem from './NotesBarListItem';
+import Folder from '@joplin/lib/models/Folder';
+import useStyles from './useStyles';
+
+interface Props {
+    themeId: number;
+	items: any[];
+	todoCheckbox_change: (checked: boolean)=> void;
+	selectedFolderId: string;
+	activeFolderId: string;
+	dispatch: any;
+	selectedNoteId: string;
+	toggleNotesBar: ()=> void;
+	settings: any;
+}
+
+function NotesBarComponent(props: Props) {
+
+	const [notes, setNotes] = React.useState<any[]>(props.items);
+	const [query, setQuery] = React.useState<string>('');
+
+	const themeId = props.themeId;
+
+	const stylingFunction = (theme: Style): Style => {
```

Comment:
Please remove this, and put all the logic under useStyle.

Reply:
Alright, noted.